lib/tracecontext/traceparent.js

```diff
@@ -0,0 +1,156 @@
+'use strict'
+
+const { randomFillSync } = require('random-poly-fill') // TODO: Remove when Node.js 6 is no longer supported
```

Comment:
Do we drop this polyfill?

Reply:
Yes, for sure. Then we can drop another unneeded dep.